SpatialGDK/Source/Public/Interop/SpatialSender.h

```diff
@@ -24,18 +24,19 @@ class USpatialReceiver;
 
 struct FPendingRPCParams
```

Comment:
Since this is referenced in both the sender and the receiver, does it not make sense to lift this out to its own file

Reply:
I personally think it's small enough to live in the sender but if everybody thinks it should move I don't mind.

Reply:
I think that would be nicer rather than having a dependency on sender in the reader but lets see what the others think

Reply:
I think lifting it into it's own file is a bit overkill but a pre-existing file might be fine. I agree with Valentyn that it's okay to live in the Sender.